src/nuxt.config.js

```diff
@@ -162,6 +168,16 @@ const nuxtConfig = {
   /**
    * Nuxt PWA modules
    */
+  workbox: {
+    runtimeCaching: [
+      {
+        urlPattern: `https://api\\.${IS_TESTNET ? 'rinkeby\\.' : '' }like\\.co/.*`,
+      },
+      {
+        urlPattern: `https://us-central1-civic-liker${IS_TESTNET ? '-develop' : '' }\\.cloudfunctions\\.net/.*`,
+      },
```

Comment:
Not regex?

Reply:
According to [doc](https://pwa.nuxtjs.org/modules/workbox.html#adding-custom-runtimecaching-items-for-cdn), the string will be compiled by `new RegExp(...)`